pkg/http/graph/resolvers/user.resolvers_test.go

```diff
@@ -401,7 +401,7 @@ func TestMutationResolver_RefreshToken(t *testing.T) {
 			expectErr: true,
 			authValidateJWTData: &http_common.MockAuthData{
 				OutputParam1: "username1",
-				OutputParam2: time.Now().Add(-time.Duration(3) * time.Minute).Unix(),
+				OutputParam2: time.Now().Add(time.Duration(3) * time.Minute).Unix(),
```

Comment:
Small but insidious bug in a test case that has driven the development of a bug.